frontend/vue/components/UserAccount/UserAccountLayout.vue

```diff
@@ -0,0 +1,138 @@
+      </div>
+      <div class="user-account__section-nav__link-list">
+        <AppLink
+          v-for="{displayName, hash} in sectionList"
+          :key="hash"
+          class="user-account__section-nav__link-list__link"
+          :class="{'user-account__section-nav__link-list__link_active': activeSection === hash}"
+          :url="hash"
+        >
+          {{ displayName }}
+        </AppLink>
+        <AppLink
+          class="user-account__section-nav__link-list__link"
+          url="/LogOut"
+        >
+          Log Out
+        </AppLink>
+      </div>
+    </nav>
+    <section class="user-account__section-container">
+      <div v-if="activeSection === sectionList[0].hash">
+        PROGRESS SECTION
+      </div>
+      <div v-if="activeSection === sectionList[1].hash">
+        GROUPS SECTION
+      </div>
+      <div v-if="activeSection === sectionList[2].hash">
+        BADGES SECTION
+      </div>
+      <div v-if="activeSection === sectionList[3].hash">
+        PRIVACY SECTION
+      </div>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import AppLink from '../common/AppLink.vue'
+
+export default defineComponent({
+  name: 'UserAccountLayout',
+  components: {
+    AppLink
+  },
+  props: {
+    userName: { type: String, required: false, default: 'Unknown' },
+    userRole: { type: String, required: false, default: 'Unknown' }
+  },
+  data () {
+    return {
+      activeSection: '',
+      sectionList: [
+        {
+          displayName: 'My Learning',
+          hash: '#MyLearning'
+        },
+        {
+          displayName: 'Groups',
+          hash: '#Groups'
+        },
+        {
+          displayName: 'Badges',
+          hash: '#Badges'
+        },
+        {
+          displayName: 'Privacy',
+          hash: '#Privacy'
+        }
+      ]
+    }
+  },
+  mounted () {
+    this.activeSection = window.location.hash || '#MyLearning'
+    window.addEventListener('hashchange', () => {
+      this.activeSection = window.location.hash
+    }, false)
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '../../../../node_modules/carbon-components/scss/globals/scss/layout';
+@import '../../../../node_modules/carbon-components/scss/globals/scss/typography';
+@import '~/../scss/variables/colors.scss';
+@import '~/../scss/variables/fonts.scss';
+
+.user-account {
+  display: flex;
+  flex-direction: row;
+  height: calc(100vh - 60px);
+
+  &__section-nav {
+    display: flex;
+    flex-direction: column;
+    width: 18rem;
+    padding: $spacing-07;
+
+    background-color: $background-color-light;
+
+    &__user-data {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: $spacing-07;
+
+      &__name {
+        @include type-style('expressive-heading-04');
+      }
+      &__role {
+        @include type-style('expressive-heading-01');
+      }
+    }
+
+    &__link-list {
```

Comment:
Goyo, I looked at the design file today and saw that at the `md` breakpoint, the width of the left-side will shrink a little...

Reply:
Ok, thank you! md breakpoint wasn't available jet when it was done :)